### ISSUE TITLE
No SSL CA with sentry-cli 1.28.3 in Docker

### ISSUE THREAD

**Issue by whs** [getsentry/sentry-cli#223]
Our CD builds are failing with this error:

```
$ sentry-cli releases deploys ${IMAGE_TAG} new -e $ENV_NAME --url $URL
error: http error: [77] Problem with the SSL CA cert (path? access rights?) (error setting certificate verify locations:
  CAfile: /etc/ssl/certs/ca-certificates.crt
  CApath: /etc/ssl/certs/)
```

Environment:

- On premises sentry
- AlphaSSL wildcard certificate
- Using docker image getsentry/sentry-cli ID=sha256:f8d2d093147106a2c1fe258cb2aad4511f27e6141827587c1ed5fe7f8fdda7a4 (1.28.3?)

**Comment by temp** [comment]
Same with 1.28.2, 1.28.1 works.

**Comment by Adzz** [comment]
+ 1, circle build failing for me with this error

**Comment by jan-auer** [comment]
Hi, thanks for reporting this issue. The musl build starting with `1.28.2` is buggy. I've just reverted the `latest` tag to `1.28.1` - please use this version in the meanwhile.

**Comment by Adzz** [comment]
Thank you! Have pinned.  `image: getsentry/sentry-cli:1.28.1`  does the job

**Comment by jan-auer** [comment]
Should be fixed in 1.28.4
